en-US/firefox/all-unified.lang

```diff
@@ -0,0 +1,143 @@
+## NOTE: https://www-dev.allizom.org/firefox/all/
+## URL: https://www-dev.allizom.org/%LOCALE%/firefox/all/
+
+
+# HTML page title. Replace "English (US)" with your local language.
```

Comment:
Passing by consideration: this is going to be a mess, and we carefully avoided going this route for the current /all page.

Reply:
I will send an email on this one - it's tricky. I am not sure how many people will read the comments.

Reply:
On L140-142 below, we have this:

```ini
# %(product_label)s is replaced with the product name e.g "Firefox", %(platform)s with the platform name e.g. "Windows", and %(locale)s with the language name e.g. "German".
;Download %(product_label)s for %(platform)s in %(locale)s
Download %(product_label)s for %(platform)s in %(locale)s
```

Couldn't this just be something like `Download the Firefox Browser in $(locale)s and more than 90 other languages` and we detect+inject the user's current locale? Or is there no way to get a "pretty name" for a locale, and we could only determine some "en-US" code?

Reply:
This is another piece I'm not happy with. Where is the language coming from? How do we ensure that is grammatically correct for the context?

Reply:
> This is another piece I'm not happy with. Where is the language coming from? How do we ensure that is grammatically correct for the context?

We already do this on the current /all page. The reason it includes both the language and platform name is for SEO.

Reply:
> HTML page title. Replace "English (US)" with your local language.

> Passing by consideration: this is going to be a mess, and we carefully avoided going this route for the current /all page.

The other option here is I could pass in the language as a variable to the string, but I don't know if that's better than leaving it up to translators to do manually. Doing this is important for SEO, so I'd like for us to try and figure out a way.

Reply:
> We already do this on the [current /all page](https://github.com/mozilla/bedrock/blob/master/bedrock/base/templates/product-all-macros.html#L87) fwiw. The reason it includes both the language and platform name is for SEO.

Interesting. That's a tooltip (not very visible), and it's wrong even in Italian (language names are not uppercase). Is this string still used in a tooltip? I guess it's broken, but if nobody cared to report bugs, it's not visible enough.

Reply:
> Is this string still used in a tooltip?

Yes it's for the same thing. Seeing as this has been this way for nearly 10 years and no one noticed, I don't think it's worth pouring time into here. If you disagree, I can remove it from the page and file a bug to fix this later.

Reply:
> Yes it's for the same thing. Seeing as this has been this way for nearly 10 years and no one noticed, I don't think it's worth pouring time into here. If you disagree, I can remove it from the page and file a bug to fix this later.

No, as said it's not worth it. But I wouldn't use the same variable/strategy in the page's title, that's going to show up.